From 2ab07feeca87e4e7d7427909eb4a546d4a9086f9 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Tue, 15 Nov 2016 01:23:50 -0500
Subject: [PATCH 49/65] pyanaconda/ui/gui/utils.py: tweak mouse arrow

The arrow of Gdk.CursorType.ARROW points to right, use
Gdk.CursorType.LEFT_PTR to replace.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase for anaconda 37.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/ui/gui/utils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/ui/gui/utils.py b/pyanaconda/ui/gui/utils.py
index 282f1bc53c..6fa2ffc3db 100644
--- a/pyanaconda/ui/gui/utils.py
+++ b/pyanaconda/ui/gui/utils.py
@@ -280,7 +280,7 @@ def unbusyCursor():
     if not window:
         return
 
-    window.set_cursor(Gdk.Cursor.new_for_display(Gdk.Display.get_default(), Gdk.CursorType.ARROW))
+    window.set_cursor(Gdk.Cursor.new_for_display(Gdk.Display.get_default(), Gdk.CursorType.LEFT_PTR))
 
 
 def ignoreEscape(dlg):
-- 
2.7.4

